prow/prowjobs/GoogleCloudPlatform/elcarro-oracle-operator/elcarro-oracle-operator.yaml

```diff
@@ -0,0 +1,21 @@
+presubmits:
+  GoogleCloudPlatform/elcarro-oracle-operator:
+  - name: operator-oss-presubmit
+    cluster: build-graybox
+    always_run: true
+    decorate: true
+    spec:
+      containers:
+      - image: gcr.io/k8s-testimages/kubekins-e2e:latest-master
+        command:
+        - make
+        args:
+        - -C
+        - oracle
+        - operator-checks
+        securityContext:
```

Comment:
We can drop the securitycontext now, this was for KIND (docker in docker) tests.